setup.py

```diff
@@ -69,4 +69,7 @@ def get_version(rel_path):
         packages=find_packages(exclude=["cloudevents.tests"]),
         version=pypi_config["version_target"],
         install_requires=["deprecation>=2.0,<3.0"],
+        extras_require={
+            "pydantic": ["pydantic"],
```

Comment:
Are you using any particular Pydantic feature that may not be available in the early versions?

Reply:
no, this works from pydantic 1.0.0

will fixate the version

Reply:
then use `>=1.0.0` here